### ISSUE TITLE
correct way to use ground truth for training and greedy for testing

### ISSUE THREAD

**Issue by yangshao** [Currie32/Text-Summarization-with-Amazon-Reviews#3]
hi, when I use a similar code to solve a seq2seq problem, during the training, the loss decrease very quickly. but during the testing, all the output sequence are same. Can you example what sample_id means?

**Comment by yangshao** [comment]
I find my problem: I forget to add <go> for the decoder input! currently it works well. Can you explain what's the meaning of logits.sample_id? why not use rnn_output? Thanks

**Comment by MaNannn** [comment]
@yangshao Hi, I meet the same problem. What is the decoder input? Thanks.

**Comment by noabenefraim** [comment]
@yangshao Can you elaborate on how you fixed this? I am having the same problem
